distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -279,7 +280,7 @@ type Table
 
              table.select_blank_columns
     select_blank_columns : Boolean -> Boolean -> Table
-    select_blank_columns self (when_any : Boolean = False) (treat_nans_as_blank : Boolean = False) =
+    select_blank_columns self (when : Boolean = False) (treat_nans_as_blank : Boolean = False) =
```

Comment:
Please update to `when` in argument list above.

Reply:
should be fixed